src/extension.ts

```diff
@@ -27,6 +27,9 @@ if (process.env.SENTRY_DSN) {
     ignoreErrors: [
       "The request failed and the interceptors did not return an alternative response",
       "ENOENT: no such file or directory",
+      "EPERM: operation not permitted",
```

Comment:
I'm not concerned about these for sidecar process starting, since we'll see `getHandlePromise(0): failed to start sidecar` errors as well.